ern-util/src/ios.js

```diff
@@ -26,7 +26,8 @@ export async function getiPhoneSimulators () {
 
 export function getiPhoneRealDevices () {
   const devices = execSync('xcrun instruments -s', {encoding: 'utf8'})
-  return parseIOSDevicesList(devices)
+  const computerName = execSync('scutil --get HostName', {encoding: 'utf8'})
```

Comment:
This `child_process` call will fail on non-macOS machines—I’m thinking CI boxes. Any ideas on how we can address?

Reply:
`ios.js` is as of now only used to support Runner for launching MiniApp(s) on a iOS device or simulator through `run-ios` command. The `run-ios` command is disabled for non macOs machine (there is a platform check). 

Doesn't make a lot of sense for CI to use `run-ios` command. In the context of system testing it might have to ideally, but we know this is a limitation for now and we can't system test `run-ios` on CI on a linux box. For unit tests we're good though, `child_process` calls can be mocked with a little effort based on the needs of the test.